server/libs/infra/src/repositories/media.repository.ts

```diff
@@ -3,6 +3,7 @@ import { exiftool } from 'exiftool-vendored';
 import ffmpeg, { FfprobeData } from 'fluent-ffmpeg';
 import sharp from 'sharp';
 import { promisify } from 'util';
+import * as fs from 'fs/promises';
```

Comment:
I don't believe we need `* as fs`. `import fs from 'fs/promises` should work just fine.